docs/get-started/get-an-allocation.md

```diff
@@ -89,6 +93,20 @@ integer value that corresponds to multiple openstack resource quotas. For exampl
     Groups** are provided to each approved projects regardless of units of requested
     quota units.
 
+### Request A New OpenShift Resource Allocation for OpenShift project
+
+![Request A New OpenShift Resource Allocation](images/coldfront-request-new-openshift-allocation.png)
+
+!!! warning "Very Important"
+    Make sure to select **NERC (OpenShift)** on Resource option and specify
+    your expected Units of computing. Be mindful, you can extend your current resource
+    allocations on your current project later on.
+
+!!! info "Information"
```

Comment:
I suggest `:information_source:` :information_source: instead of !!! as this is information not warning.

Reply:
Never mind I see that your markdown to html changes !!! info to something else.